arangod/Cluster/ClusterComm.cpp

```diff
@@ -38,10 +38,54 @@
 #include "VocBase/ticks.h"
 
 #include <thread>
+#include <iomanip>
 
 using namespace arangodb;
 using namespace arangodb::communicator;
 
+
+namespace {
+std::stringstream createRequestInfo(NewRequest const& request) {
+  bool trace = Logger::CLUSTERCOMM.level() == LogLevel::TRACE;
+  std::stringstream ss;
+  ss << "id: " << std::setw(8) << std::setiosflags(std::ios::left)
+               << request._ticketId << std::resetiosflags(std::ios::adjustfield)
+     << " --> " << request._destination
+     << " -- " << ( request._request->fullUrl().empty() ? "url unkown" : request._request->fullUrl())
```

Comment:
```suggestion
     << " -- " << ( request._request->fullUrl().empty() ? "url unknown" : request._request->fullUrl())
```

Reply:
I think it would make sense to also show the HTTP method used (e.g. GET, POST, PUT) here.